config/development_cass-es.yaml

```diff
@@ -89,16 +95,15 @@ archival:
 namespaceDefaults:
   archival:
     history:
-      state: "enabled"
+      state: "disabled"
       URI: "file:///tmp/temporal_archival/development"
     visibility:
-      state: "enabled"
+      state: "disabled"
       URI: "file:///tmp/temporal_vis_archival/development"
 
 publicClient:
   hostPort: "localhost:7233"
 
 dynamicConfigClient:
   filepath: "config/dynamicconfig/development_es.yaml"
-  pollInterval: "10s"
-
```

Comment:
Previously when I do `--zone es start`, this file will be merged with `development.yaml` and result in using dual visibility manager, which is super confusing... 

I like the new approach, just not sure if this will be surprising for others who is currently using dual manager setup.